pages.zh/linux/mkfs.btrfs.md

```diff
@@ -0,0 +1,17 @@
+# mkfs.btrfs
+
+> 创建一个BTRFS文件系统。
+> 默认情况下是 `raid1`，指定了数据块的两份拷贝分布在两个不同的设备上。
+> 更多信息： <https://btrfs.readthedocs.io/en/latest/mkfs.btrfs.html>.
```

Comment:
A general copywriting rule in Chinese is adding a space between full-width and half-width characters.
```suggestion
> 创建一个 BTRFS 文件系统。
> 默认情况下是 `raid1`，指定了数据块的两份拷贝分布在两个不同的设备上。
> 更多信息：<https://btrfs.readthedocs.io/en/latest/mkfs.btrfs.html>.
```

Reply:
Thanks for reviewing!